server/routes/__tests__/money.spec.js

```diff
@@ -154,7 +153,7 @@ describe('Prisoner Money', () => {
         if (requestUrl.match(/\/transaction-history/i)) {
           return Promise.resolve(transactionApiResponse);
         }
-        if (requestUrl.match(/\/agencies\/prison/i)) {
+        if (requestUrl.match(/\/agencies\/type\/INST\?activeOnly=false/i)) {
```

Comment:
Inherited I appreciate, but the replication of the regexp, kinda suggests that it might be better as a const or wrapped in a descriptive function 🤷‍♀️ Probably a code style thing...

Reply:
this needs tidying up generally